helpers.go

```diff
@@ -870,6 +885,55 @@ func (ctx *e2econtext) summarizeSuiteFindings(t *testing.T, suite string) {
 		}
 		t.Logf("Scan %s contained %d checks that failed, but have a remediation available", scan.Name, len(failedWithRemediationList.Items))
 	}
+	// print out the profile assertion file if it doesn't exist
+	if !ctx.profileAssertExist && ctx.profileAssert.RuleResults != nil {
+		profileTestBytes, err := yaml.Marshal(ctx.profileAssert)
+		if err != nil {
+			t.Fatalf("failed to marshal profile assertion file: %s", err)
+		}
+		t.Logf("Profile assertion file:\n%s", string(profileTestBytes))
+	}
+}
+
+func (ctx *e2econtext) assertProfileAssertionFile(t *testing.T, resultList *cmpv1alpha1.ComplianceCheckResultList, afterRemediations bool) {
+	profileTestFilePath := path.Join(platformTestFolderPath, ctx.product+"-"+ctx.Profile+"-"+ctx.version+".yml")
```

Comment:
I wasn't able to get this working locally due to the suite not being able to find the assertion files. I think it's because this path isn't taking into account the rootdir.

Reply:
Also - the `platform` is always '' when this is evaluated, which might be why it's looking in the wrong place. 

```
No profile assertion file found, you should create /home/lbragstad/Projects/content/tests/assertions/ocp4-cis-4.15.yml to assert the test results
```